content/en/docs/contribute/localization.md

```diff
@@ -183,15 +183,17 @@ To collaborate on a development branch:
 
 3. Approvers review and merge feature branches into the development branch.
 
-4. Periodically, an approver merges the development branch to its source branch.
+4. Periodically, an approver merges the development branch to its source branch, usually by opening and approving a new pull request. We recommend to squash the commits in [a single commit with multiple authors](https://help.github.com/en/articles/creating-a-commit-with-multiple-authors) before approving the pull request.
 
 Repeat steps 1-4 as needed until the localization is complete. For example, subsequent German development branches would be: `dev-1.12-de.2`, `dev-1.12-de.3`, etc.
 
 Teams must merge localized content into the same release branch from which the content was sourced. For example, a development branch sourced from {{< release-branch >}} must be based on {{< release-branch >}}.
 
 An approver must maintain a development branch by keeping it current with its source branch and resolving merge conflicts. The longer a development branch stays open, the more maintenance it typically requires. Consider periodically merging development branches and opening new ones, rather than maintaining one extremely long-running development branch.
 
-While only approvers can merge pull requests, anyone can open a pull request for a new development branch. No special permissions are required.
+On the beginning of every team milestone, a team member usually opens an issue to update outdated contents by comparing upstream changes between the previous development branch and the current development branch.
```

Comment:
@gochist Do you have a gist of the script you use for comparison? Would you please provide a link?

Reply:
@zacharysarah Here is the link, https://gist.github.com/gochist/8654460717b4bbb4ac490dda79a547fa